script.js

```diff
@@ -0,0 +1,62 @@
+const song = document.querySelector('.song');
+const play = document.querySelector('.play');
+const outline = document.querySelector('.moving-outline circle');
+const video = document.querySelector('.video-container video');
+const sounds = document.querySelectorAll('.container__sounds button');
+const timeDisplay = document.querySelector('.time-display');
+const timeSelect = document.querySelectorAll('.time-select');
+const outlineLenght = outline.getTotalLength();
+const ZERO = '0';
+let fakeTime = 600;
+
+outline.style.strokeDasharray = outlineLenght;
+outline.style.strokeDashoffset = outlineLenght;
+
+const checkPlaying = song => {
+  if(song.paused) {
+    song.play();
+    play.src = 'svg/pause.svg';
+    video.play();
+  }else {
+    song.pause();
+    play.src = 'svg/play.svg';
+    video.pause();
+  }
+};
+
+play.addEventListener('click', () => checkPlaying(song));
+
+function addZero(n) {
+  return (parseInt(n, 10) < 10 ? ZERO : '') + n;

```

Comment:
What is 10? Use constants with the corresponding name.

Reply:
DONE!